<launch>

    <arg name="markerId"        default="26"/>
    <arg name="markerSize"      default="0.08"/>    <!-- in m -->
    <!-- <arg name="eye"             default="left"/> -->
    <arg name="marker_frame"    default="aruco_marker_frame"/>
    <arg name="ref_frame"       default="bebop2/base_link"/>  <!-- leave empty and the pose will be published wrt param parent_name -->
    
    <arg name="corner_refinement" default="LINES" /> <!-- NONE, HARRIS, LINES, SUBPIX -->


    <node pkg="aruco_ros" type="single" name="aruco_single">
        <remap from="/camera_info" to="bebop2/camera_base/camera_info" />
        <remap from="/image" to="/bebop2/camera_base/image_raw" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="marker_id"          value="$(arg markerId)"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>   <!-- frame in which the marker pose will be refered -->
        <param name="camera_frame"       value="bebop2/camera_base_optical_link"/>
        <param name="marker_frame"       value="$(arg marker_frame)" />
        <param name="corner_refinement"  value="$(arg corner_refinement)" />
    </node>
    <!-- <node pkg="Interface_Bebop" type="RobotPoseInWorld.py" name="RobotPoseInWorld">
    </node> -->


</launch>

